.github/workflows/antora.yml

```diff
@@ -53,7 +61,7 @@ jobs:
         working-directory: ${{ inputs.path }}
 
       - name: Publish to GitHub Pages
-        if: ${{ inputs.publish }}
+        if: ${{ inputs.publish && github.event.repository.fork == false }}
```

Comment:
Why not for forks? If someone wants to fork and put a doc online, should that not be possible? Or maybe I just don't understand.

Reply:
We can always change this if someone needs it. But it doesn't seems to me necessary to anyone, including ourselves. 😄 
I did this by analogy with our artifact publishing workflow
https://github.com/playframework/.github/blob/7fc2a9ae0c3b3971ede89fe3b6b9f88183d0d8ee/.github/workflows/publish.yml#L19